PythonTool/DataLoading.py

```diff
@@ -0,0 +1,208 @@
+import argparse
+import visit as Vi
+from GraveIdentifyAndRemove import remove_graveyard
+from pymoab import core, types, tag
+
+
+def parse_arguments():
+    """
+    Parse the argument list and return the location of a geometry file, the
+    location of a data file, and indication if the user wants images of the
+    four default plot windows saved or not in the current directory.
+
+    Input:
+    ______
+       none
+
+    Returns:
+    ________
+       args: Namespace
+           User supplied geometry file location, data file location, and
+           indication if the user wants images of the plot windows saved.
+    """
+
+    parser = argparse.ArgumentParser(description="Create default VisIt output.")
+
+    parser.add_argument("geofile",
+                        type=str,
+                        help="Provide a path to the geometry file."
+                        )
+    parser.add_argument("datafile",
+                        type=str,
+                        help="Provide a path to the data file."
+                        )
+
+    parser.add_argument("-i", "--images",
+                        action="store_true",
+                        help="Indicate whether or not to save images of plot windows."
+                        )
+
+    args = parser.parse_args()
+
+    return args
+
+
+def py_mb_convert(file_location, file_extension):
+   """
+   Convert files from one format to another with PyMOAB.
+
+   Input:
+   ______
+      file_location: str
+          User supplied file location.
+      file_extension: str
+          User supplied file format to convert to, including '.'
+
+   Returns:
+   ________
+      new_file_name: str
+          The new file name with the specified extension.
+   """
+
+   # Load the file to be converted.
+   mb = core.Core()
+   mb.load_file(file_location)
+
+   # Isolate file name from string containing the file location.
+   input_array = file_location.split("/")
+   input_file = input_array[-1]
+   base_file_name = input_file[:-4]
+   new_file_name = base_file_name + file_extension
+
+   # Write the new file with the user supplied extension.
+   mb.write_file(new_file_name)
+
+   return(new_file_name)
```

Comment:
The more commonly used syntax for returns of a single variable would be: `return new_file_name`